drivers/include/bmx055.h

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_bmx055 BMX055 9-axis sensor
+ * @ingroup     drivers_sensors
+ * @brief       Device driver for the BMX055 9-axis sensor
+ * @{
+ *
+ * @file
+ * @brief       Device driver interface for the BMX055 9-axis sensor
+ *
+ * @note        The current state of the driver only implements basic polling.
+ *
+ * @author      Semjon Kerner <semjon.kerner@fu-berlin.de>
+ */
+
+#ifndef BMX055_H
+#define BMX055_H
+
+#include <stdint.h>
+
+#include "periph/i2c.h"
+#include "periph/gpio.h"
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   The sensors default I2C addresses
+ */
+#define BMX055_MAG_ADDR_DEFAULT  0x10
+#define BMX055_ACC_ADDR_DEFAULT  0x18
```

Comment:
Those 2 defines are not documented. Use `@name` around the 3 or a `@brief` for each ones